site/docs/master/httpproxy.md

```diff
@@ -715,7 +715,65 @@ spec:
           port: 80
 ```
 
-#### ExternalName
+#### Path Rewriting
+
+HTTPProxy supports rewriting the HTTP request URL path prior to delivering the request to the service.
+Rewriting is performed after a routing decision has been made, and never changes the request destination.
+
+The `pathRewritePolicy` field specifies how the path prefix should be rewritten.
+The `replacePrefix` rewrite policy specifies a replacement string for a HTTP request path prefix match.
+When this field is present, the path prefix that the request matched is replaced by the text specified in the `replacement` field.
+The original path before rewriting will be placed into the into the `x-envoy-original-path` header.
+
+```yaml
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+metadata:
+  name: rewrite-example
+  namespace: default
+spec:
+  virtualhost:
+    fqdn: rewrite.bar.com
+  routes:
+  - services:
+    - name: s1
+      port: 80
+    pathRewritePolicy:
+      replacePrefix:
+      - replacement: "/new/prefix"
```

Comment:
for consistency does this need to be quoted? fqdn, name and namespace were not

Reply:
Fixed.